client/package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "^4.3.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "antd": "^4.8.0",
+    "prop-types": "^15.7.2",
```

Comment:
NIT: This is nice to have but would only be effective if we enforce proptypes checking on all components. Not sure if that's the direction that we want to move forward but I guess we can keep it here for now?

Reply:
I guess it’s up to people to add it; but I personally would. Makes it easier to understand what each prop is